components/loading_image_preview.tsx

```diff
@@ -25,26 +30,10 @@ export default function LoadingImagePreview({loading, progress, containerClass})
             {progressView}
         </div>
     );
-}
-
-LoadingImagePreview.propTypes = {
-
-    /**
-     * The percent number of the progress
-     */
-    progress: PropTypes.number,
-
-    /**
-     * The loading message to display
-     */
-    loading: PropTypes.string,
-
-    /**
-     * Loading style
-     */
-    containerClass: PropTypes.string,
 };
 
 LoadingImagePreview.defaultProps = {
     containerClass: 'view-image__loading',
 };
+
+export default LoadingImagePreview;
```

Comment:
How come the export has been moved down here?

Reply:
Hey @devinbinnie, I had moved it when I declared the function as React.FC<Props>, would you like me to change it back? I was following what I had seen other TS functional components were doing in the application (eg: https://github.com/mattermost/mattermost-webapp/blob/master/components/widgets/admin_console/admin_panel_togglable.tsx). I can revert it if you'd like!

Reply:
If it's needed to declare as React.FC then that's okay.